src/config/Pyegi/PythonScripts/Disintegration/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "Buttons": ["Apply", "Alaki", "Cancel"],
+  "Buttons": ["Apply", "Alaki", "Back", "Cancel"],
```

Comment:
now that I'm thinking about these, I think there is no point to receive buttons from developer. because we don't have support for custom buttons. I think we can delete this section from all settings.json. when we wanted to support custom guis, then we can bring this options back. but for now, there is no points in receiving these. We can just have `Apply`, `Back` and `Cancel` for all scripts.